src/callbacks/evaluate_progress.cpp

```diff
@@ -81,7 +83,8 @@ EvalType evaluate(model& m, const std::string& metric_name, execution_mode mode)
   // Clean up and return metric value
   m.reset_mode(c, original_mode);
   dc.reset_mode(c);
-  return metric_value;
+  return -1.0;
+  // return metric_value;
```

Comment:
Following on to above suggestions:
```suggestion
  return metric_value;
```

If you don't like these suggestions, please just delete the code and change the return type of the function -- it lives in the anonymous namespace, so I can't imagine that will be too much of a headache.